server/tests/models/record.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable max-len */
+
+import { Record } from '../../src/models/record.js';
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+
+describe('Testing Record model', function() {
+  it('1. Creating new record', function(done) {
+    const r = new Record({id: 'r1', latitude: -87.46, longitude: 180, timestamp: new Date('2022-02-22T10:08:00.000-05:00'), driver_id: 'd1', slip_score: 50.88});
+    r.validate().then(() => {
+      expect(r.id).to.equal('r1');
+      expect(r.latitude).to.equal(-87.46);
+      expect(r.longitude).to.equal(180);
+      expect(r.timestamp.valueOf()).to.equal(new Date(1645542480000).valueOf());
```

Comment:
The test for timestamp is a bit fragile and could break if the timezone of the test environment changes. Consider using a library like moment.js to parse and compare timestamps to avoid potential issues with timezones.

Reply:
There shouldn't be any issue with time zones, because neither the epoch timestamp nor the earlier line initialized from date with time zone offset specified depends on the actual time zone of the device the test is running on.